src/fhir/upload.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python3
+
+import json
+import os
+import requests
+import argparse
+
+
+def setup_args():
+    # Usage message is the module's docstring.
+
+    parser = argparse.ArgumentParser(epilog=__doc__)
+    parser.add_argument("--verbose", action="store_true", help="increase verbosity")
+    parser.add_argument(
+        "fhir_server_url", action="store", nargs=1, help="FHIR server base URL"
+    )
+    parser.add_argument(
+        "fhir_files",
+        action="store",
+        nargs="*",
+        default=None,
+        help="fhir JSON files to upload",
+    )
+    args = parser.parse_args()
+    return args
+
+
+def find_files(path):
+    """Find all possible FHIR files in the script path"""
+    for fname in (
+        fname for fname in os.scandir(path) if fname.name.lower().endswith(".json")
+    ):
+        yield fname
+
+
+def bundle_files(fhir_resources):
+    """Combine individual FHIR resources into a single transaction Bundle"""
+    tx_bundle = {
+        "resourceType": "Bundle",
+        # "id": "TBD",
+        "type": "transaction",
+        "entry": [],
+    }
+
+    for fhir_resource in fhir_resources:
+        if "id" not in fhir_resource:
+            print("skipping: ", fhir_resource)
```

Comment:
perhaps this is just debugging, but it might be nice to know why it was skipped (i.e. no `id` attribute)